docs/recipes/babelrc.md

```diff
@@ -4,50 +4,77 @@ Translations: [Français](https://github.com/sindresorhus/ava-docs/blob/master/f
 
 There are multiple options for configuring how AVA transpiles your tests using Babel.
 
- - [Specify a complete config in `package.json`](#specify-a-complete-config-in-packagejson)
- - [Extend existing `.babelrc` without modification](#extend-existing-babelrc-without-modification)
- - [Extend existing `.babelrc` with additional plugins or presets](#extend-existing-babelrc-with-additional-plugins-or-presets)
- - [Extend an alternate config file (i.e. not `.babelrc`)](#extend-alternate-config-file)
+ - [AVA's default transpiler behavior](#avas-default-transpiler-behavior)
+ - [Customizing how AVA transpiles your tests](#customizing-how-ava-transpiles-your-tests)
+ - [Transpiling Sources](#transpiling-sources)
+ - [Transpiling tests and sources the same way](#transpiling-tests-and-sources-the-same-way)
+ - [Extend your source transpilation configuration](#extend-your-source-transpilation-configuration)
+ - [Extend an alternate config file (i.e. not `.babelrc`)](#extend-an-alternate-config-file)
  - [Notes](#notes)
 
-## Specify a complete config in `package.json`
+## AVA's default transpiler behavior
 
-The `babelrc` option defaults to `false`, meaning `.babelrc` files are not considered when transpiling tests. This means you must specify your complete config in `package.json`.
+By default, AVA transpiles your tests (and only your tests) using Babels [`es2015`](http://babeljs.io/docs/plugins/preset-es2015/) and [`stage-2`](http://babeljs.io/docs/plugins/preset-stage-2/) presets. This is a great option for small modules where you do not desire a build step to transpile your source before deploying to `npm`.
+
+## Customizing how AVA transpiles your tests
+
+You can override the default Babel configuration AVA uses for test transpilation in `package.json`. For example, the configuration below adds the Babel `rewire` plugin, and opts to only use the Babel [`stage-3`](http://babeljs.io/docs/plugins/preset-stage-3/) preset (which is a subset of [`stage-2`]((http://babeljs.io/docs/plugins/preset-stage-2/))).
```

Comment:
I mostly read raw markdown documents and having to hunt for matching links at the bottom gets annoying fast.